lib/routes/pianyuan/utils.js

```diff
@@ -0,0 +1,89 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const url = require('url');
+const config = require('@/config').value;
+const security_key = 'pianyuan-security_session_verify';
+const PHPSESSID_key = 'pianyuan-PHPSESSID';
+const loginauth_key = 'pianyuan-py_loginauth';
+
+const ProcessFeed = async (list, cache) => {
+    const link_base = 'https://pianyuan.org';
+
+    const items = await Promise.all(
+        list.map(async (e) => {
+            const link = url.resolve(link_base, e);
```

Comment:
`url.resolve` is deprecated. Please use [`URL`](https://nodejs.org/api/url.html#the-whatwg-url-api) instead.